_editor/plugins/LinkDialog.js

```diff
@@ -431,6 +432,34 @@ define([
 					}, 10);
 				}
 			}
+		},
+
+		_onMouseUp: function(){
+			// summary:
+			//		Function to define a behavior on mouse up on the element
+			//		type this dialog edits to move the cursor just outside
+			//		anchor tags when clicking on their edges.
+			// tags:
+			//		protected.
+			if(has('ff')){
```

Comment:
I assume there's no better way to detect this and a check for Firefox as a whole.

Reply:
I'd be happy to have this work across the board because, as I mentioned above, even Chrome has situations where it leaves the cursor inside the anchor. Someone with more expertise may be able to do the right feature detection to ensure these APIs function as expected across the board.